design-documents/service-isolation/inter-service-communication.md

```diff
@@ -0,0 +1,301 @@
+### Inter-service communications
```

Comment:
Another idea someone mentioned is making service a demon. This would allow save time on application bootstrap.